pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/JsonAsyncHttpPinotClientTransport.java

```diff
@@ -84,11 +89,12 @@ public JsonAsyncHttpPinotClientTransport(Map<String, String> headers, String sch
     _httpClient = Dsl.asyncHttpClient(builder.build());
   }
 
-  public JsonAsyncHttpPinotClientTransport(Map<String, String> headers, String scheme,
+  public JsonAsyncHttpPinotClientTransport(Map<String, String> headers, String scheme, String extraOptionStr,
     @Nullable SslContext sslContext, ConnectionTimeouts connectionTimeouts, TlsProtocols tlsProtocols) {
     _brokerReadTimeout = connectionTimeouts.getReadTimeoutMs();
     _headers = headers;
     _scheme = scheme;
+    _extraOptionStr = StringUtils.isEmpty(extraOptionStr) ? DEFAULT_EXTRA_QUERY_OPTION_STRING : extraOptionStr;
```

Comment:
don't we ned to concat the DEFAULT with what user passed in to keep behavior same ?

Reply:
yeah I thought about it and wasn't sure if this is a good idea since these 2 options have been in the deprecation process for a while. but good call out

Reply:
Ohh yeah. Let's probably ignore then